Debugging.md

```diff
@@ -0,0 +1,83 @@
+# Debugging your Game Servers
+
+Many debugging tasks of PlayFab Multiplayer Servers can be done locally using [LocalMultiplayerAgent](https://github.com/PlayFab/LocalMultiplayerAgent). However, sometimes the local environment does not exactly match the actual service and you would like to connect directly to debug a running server because it is marked as unhealthy or not performing as expected.
+
+In order to connect to the VM hosting your game server (either Windows or Linux), you can get RDP/SSH credentials using the "Connect" button on playfab.com web application (you can see this button on the "Virtual Machines" page on your Multiplayer Build) or using [CreateRemoteUser](https://docs.microsoft.com/en-us/rest/api/playfab/multiplayer/multiplayerserver/createremoteuser?view=playfab-rest) API call.
+
+As soon as you connect to the VM, you can use the console of the operating system to monitor your game servers. MPS service is natively using Docker containers on both Windows and Linux to spin up game server processes. To run Docker CLI commands, you'll need an admin powershell on Windows and `sudo su -` on Linux.
+
+**These are advanced Docker container debugging instructions, usage of them might break your game servers. You should not use these on servers handling retail traffic until you are very familiar with docker?**.
```

Comment:
>You [](start = 107, length = 3)

We should just say "We do not recommend using these commands on containers running your production game servers." No need to mention whether they are familiar or not